src/home_robot/home_robot/manipulation/grasping.py

```diff
@@ -52,6 +52,10 @@ def plan_to_grasp(
             print("-> could not solve for grasp")
             return None
 
+        # Overwrite standoff pos.z with a really high value so it comes in from above
+        standoff_pos = grasp_pos + np.array([0.0, 0.0, 0.4])
+        standoff_pos[2] = np.min([1.2, standoff_pos[2]])
```

Comment:
is this fixed in main?

Reply:
I think max_arm_height could still go up by 0.2. With 1.2 as current height, we can't pick things from drawer without hitting its top edge.

Reply:
To be clear, I am suggesting `max_arm_height = 1.4`